Header header
bool is_landed
bool is_waiting_for_command